src/android/CodePushException.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.cordova;
+
+/**
+ * Created by kgoranchev on 19/10/2017.
```

Comment:
Please remove auto generated comments, don't use it throughout the project

Reply:
Will do.